components/studio/test/shared/studio-exit/test-studio-umount-long-fs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash 
+
+set -euo pipefail 
+
+sudo hab pkg install core/e2fsprogs
+
+# Maximum directory name length is 255 characters so we need to create
+# a nested set of directories to have a mount point with > 1024 characters. 
+tmpdir=$(mktemp -d -p /tmp hab-studio-XXXXXXX)
+directory="$(printf "a%.0s" {1..100})"
+mnt_path="/mnt/$(printf "$directory/%.0s" {1..10})"
+
+cleanup() { 
+  sudo umount "$mnt_path" || true
+  sudo rm -rf "$mnt_path"
+  ( cd "$tmpdir"/studio && $STUDIO_COMMAND rm )
```

Comment:
I prefer the argument-style invocation as well,  this was done for consistency with how the expect `studio-enter.exp` script is invoked. Since the argument is one or two args, depending on what we're testing (`hab studio` vs `/path/to/hab-studio.sh`) the env var was a much more straight forward solution.  

I'm doing some investigation to see if I can make it work as an argument to the expect script, that would be preferred. If not, I _think_ I would like to keep consistency in how we invoke the tests.

Reply:
I'll leave it to your discretion